sdk/python/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb

```diff
@@ -87,7 +87,10 @@
      "transient": {
       "deleting": false
      }
-    }
+    },
+    "tags": [
+     "validation-workspace"
+    ]
```

Comment:
for my understanding how are these tags (referring to all of them not these in particular) used?

Reply:
discussed offline